packages/gatsby/content/getting-started/questions-and-answers.md

```diff
@@ -171,44 +171,44 @@ Should you want to run a script in the base of your project:
 
 **No.**
 
-As mentioned in the [previous section](#why-registryyarnpkgcom-does-facebook-track-us), the Yarn registry is just a CNAME to the npm registry. Since we don't even have a backend, any server error can only come from the npm registry and thus should be reported to them and monitored on their [status page](https://status.npmjs.org/).
+As mentioned in the [previous section](#why-registryyarnpkgcom-does-facebook-track-us), the Yarn registry is just a CNAME to the npm registry. Since we don't even have a backend, any server error comes from the npm registry and thus should be reported to them and monitored on their [status page](https://status.npmjs.org/).
 
 ## Is Yarn faster than other package managers?
 
 **Shrug 🤷‍♀️**
 
-At the time Yarn got released Yarn was effectively much faster than some of its competitors. Unfortunately, we failed to highlight that performance wasn't the main reason why we kept working on Yarn. Performances come and go, so while we were super fast it wasn't so much because we were doing something incredibly well, but rather that the competing implementations had a serious bug. When that bug got fixed, our miscommunication became more apparent as some people thought that Yarn was all about performances.
+Yarn was much faster than some of its competitors when it was first released. Unfortunately, we failed to highlight that performance wasn't the main reason why we kept working on Yarn. Performance is an ongoing battle for most projects though, and Yarn's speed wasn't so much because we were doing something incredibly well, but rather the competing implementations had issues that seriously impacted their performance. As those issues were resolved, our miscommunication became more apparent as some people thought that Yarn was all about performances.
 
-Put simply, our differences lie in our priorities. Different projects make different tradeoffs, and it's exactly what happens here. We prioritized workspaces because we felt like monorepos were providing significant value. We've spent significant resources pushing for Plug'n'Play (including through [dozens of contributions to third-party projects](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Aarcanis+archived%3Afalse+is%3Aclosed+pnp+-user%3Ayarnpkg+)) because we felt like this was important for the ecosystem. This is the main difference: we make our own informed decisions regarding the project roadmap.
+Put simply, different projects make different tradeoffs.  We've spent significant resources pushing for Plug'n'Play (including through [dozens of contributions to third-party projects](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Aarcanis+archived%3Afalse+is%3Aclosed+pnp+-user%3Ayarnpkg+)) because we felt like this was important for the ecosystem. We prioritized workspaces because we felt like monorepos were providing significant value. This is the main difference: we make our own informed decisions regarding the project roadmap.
```

Comment:
```suggestion
Put simply, different projects make different tradeoffs. We've spent significant resources pushing for Plug'n'Play (including through [dozens of contributions to third-party projects](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Aarcanis+archived%3Afalse+is%3Aclosed+pnp+-user%3Ayarnpkg+)) because we felt like this was important for the ecosystem. We prioritized workspaces because we felt that monorepos provided significant value. This is Yarn's main distinction: we make our own informed decisions regarding the project roadmap.
```